### DISCUSSION TITLE
Go to pdf preview section from latex code

### DISCUSSION THREAD

**Discussion by picatostas** [James-Yu/LaTeX-Workshop#3390] [Q&A] [answered]
### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before requesting a feature.

You visited the wiki?: Yes

### If your requested feature is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Is your requested feature  related to a problem? No

A clear and concise description of what the pro
## Describe the solution you'd like  [Required]

I wonder if there is a way to go to the preview in the pdf document from the editor for the section that you are currently editing or about to edit. I took a look at the available commands/shortcuts and I cannot find it. The other way around works as expected, I will will 'Ctrl + Left Click' in the pdf document and it will take to the latex code for that particular section. Any help would be much appreciated!

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Windows 11 Home 22000.795
 - VS Code version: 1.69.2
 - Extension version: v8.27.2
 - TeX distribution version: Tex Live 2022

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

**Comment by James-Yu** [comment, marked as answer]
That is called forward synctex. Check the TeX badge in the sidebar for this command.

**Reply by picatostas** [reply]
That's great ! Thanks a lot for your prompt reply.

**Comment by ricardoavelino** [comment]
Thanks for the amazing package @James-Yu. I was wondering if there's a way to set a shortcut to this "SyncTeX from cursor" function.

Also, is it possible to do the inverse, i.e., click on the .pdf paragraph text and find the corresponding .tex location?

**Reply by SirHamburger** [reply]
This should work with Ctrl+left click on the PDF.
